geode-core/src/main/java/org/apache/geode/internal/lang/SystemPropertyHelper.java

```diff
@@ -113,6 +113,14 @@
    */
   public static final String RE_AUTHENTICATE_WAIT_TIME = "reauthenticate.wait.time";
 
+  /**
+   * Maximum number of concurrent executions in a server of wan-copy region commands.
+   * Once the maximum number is reached, subsequent executions will be halted until
+   * a thread for any of the ongoing executions is released.
+   */
+  public static final String WAN_COPY_REGION_FUNCTION_MAX_CONCURRENT_THREADS =
```

Comment:
Just to make sure that we don't get the property wrong, I would want to make sure that this property is `geode.wan.copy-region.max-threads`. So we might require a few more contextual property "levels".

@upthewaterspout @pivotal-jbarrett what are your thoughts here? I know that we've never really formally discussed contextual property hierarchies, but is this something that we might consider starting here?